src/app/components/MainContent/index.jsx

```diff
@@ -24,7 +24,7 @@ const render = ({ blocks }) =>
   });
 
 const MainContent = ({ data }) => {
-  const renderedContent = render(data);
+  const renderedContent = render(data.model);
```

Comment:
See my comment [here](https://github.com/bbc/simorgh/pull/247/files#diff-5f296fd8afe33a8853c3f2655172c257R43), this shouldn't need to change.